save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FileUploader.kt

```diff
@@ -220,28 +221,33 @@ fun <F : AbstractFileInfo> fileUploader() = FC<UploaderProps<F>> { props ->
     }
 
     val (fileForUploading, setFileForUploading) = useState<File>()
+    @Suppress("TOO_MANY_LINES_IN_LAMBDA")
     val uploadFile = useDeferredRequest {
-        fileForUploading?.let {
-            val response = post(
-                props.getUrlForFileUpload(),
-                Headers(),
-                FormData().apply {
-                    append("file", fileForUploading)
-                },
-                loadingHandler = ::noopLoadingHandler,
-            )
-                .let {
-                    props.decodeFileInfoFromString(it)
-                }
+        fileForUploading?.let { fileForUploading ->
+            if (fileForUploading.name != FileType.SETUP_SH.fileName || !props.isSandboxMode) {
```

Comment:
suggestion to flip conditions: `props.isSandboxMode` is more light checking

Reply:
Done